meeting-search.php

```diff
@@ -48,21 +44,19 @@
             echo "<Sms>" . word('there_are_no_other_meetings_for_today') . "</Sms>";
         }
     }
-
     $results_counter = 0;
     for ($i = 0; $i < count($filtered_list); $i++) {
         $results = getResultsString($filtered_list[$i]);
-
         if (!isset($_REQUEST["SmsSid"])) {
             echo "<Pause length=\"1\"/>";
             echo "<Say voice=\"" . setting('voice') . "\" language=\"" . setting('language') . "\">" . word('number') . " " . ($results_counter + 1) . "</Say>";
-            echo "<Say voice=\"" . setting('voice') . "\" language=\"" . setting('language') . "\">" . $results[0] . "</Say>";
-            echo "<Pause length=\"1\"/>";
+	    echo "<Say voice=\"" . setting('voice') . "\" language=\"" . setting('language') . "\">" . $results[0] . "</Say>";
+	    echo "<Pause length=\"1\"/>";
             echo "<Say voice=\"" . setting('voice') . "\" language=\"" . setting('language') . "\">" . word('starts_at') . " " . $results[1] . "</Say>";
-            echo "<Pause length=\"1\"/>";
-            echo "<Say voice=\"" . setting('voice') . "\" language=\"" . setting('language') . "\">" . $results[2] . "</Say>";
+	    echo "<Pause length=\"1\"/>";
+	    $results[2] = preg_replace($city_pron_fix_a, $city_pron_fix_b, $results[2]);
```

Comment:
This seems somewhat limiting in how the replacement is done.  The way you have implemented this would only handle a single replacement.

Reply:
I concur with you.  This was just all I could find.  You could do more than one replacement but it would require duplication of everything each time.  Not a problem for one or two but would be a pain for very many. There's still the issue of the text messages taking on the pronunciation spelling.  I was just trying to pronounce Bossier correctly which it did.